react-app-drawer-component/my-app/src/AppDrawer.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import { MdMenu } from 'react-icons/md';
+
+type Props = {
+  header: string[];
+};
+
+export function AppDrawer({ header }: Props) {
+  const [opened, setOpened] = useState(false);
+  const [updateTitle, setUpdateTitle] = useState('No Selected Title');
+
+  return (
+    <div className={opened ? 'overlay' : 'no-overlay'}>
+      <MenuButton menuClick={() => setOpened(!opened)} />
+      <div
+        onClick={() => setOpened(!opened)}
+        className="flex row-to-column center-all">
+        <h1>{updateTitle}</h1>
+        <svg
```

Comment:
You didn't want to just import `hylian-emblem.svg`? (If you move it to your src folder you can import it.)